pipelines/LSSTCam/cpFlat.yaml

```diff
@@ -2,3 +2,13 @@ description: cp_pipe LSSTCam flat calibration construction.
 instrument: lsst.obs.lsst.LsstCam
 imports:
   - location: $CP_PIPE_DIR/pipelines/_ingredients/cpFlatLSST.yaml
+tasks:
+  cpFlatCombine:
+    class: lsst.cp.pipe.CalibCombineByFilterTask
+    config:
+      doVignetteMask: true
+      # The radius is derived from the Batoid vignetting
+      # polynomial, with a threshold of 0.1 (90% vignetted).
+      # This corresponds to roughly 80% vignetting in the dome
+      # flats.
+      vignette.radius: 364.51
```

Comment:
Do we know why there's a difference between Batoid and the dome flats?  Idealized perfect beams compared to a reality with complicated stray light?  I don't think it actually matters, but this is a larger difference than I would have naively expected.

Reply:
The batoid vignetting function was a polynomial fit to the focused light.  The flat tracks focused + scattered light (roughly along with the background) so I'm not surprised there's a difference here especially so far out where it's falling off very rapidly ... and I don't know how good a fit the Batoid polynomial was at the far radii.